docs/cli/nrfu.md

```diff
@@ -0,0 +1,94 @@
+# Execute NRFU testing
+
+All the NRFU testing commands are placed under `anta nrfu` and provides different rendering options:
+
+- Table view
+- JSON view
+- Text view
+
+```bash
+anta nrfu
+Usage: anta nrfu [OPTIONS] COMMAND [ARGS]...
+
+  Run NRFU against inventory devices
+
+Options:
+  --help  Show this message and exit.
+
+Commands:
+  json   ANTA command to check network state with JSON result
+  table  ANTA command to check network states with table result
+  text   ANTA command to check network states with text result
+```
+
+All of these commands require the following input:
+
+- A path to a catalog of tests to execute (`--catalog`)
+- A list of tags if they are part of your inventory (`--tags`). List is comma separated
+
+
+## NRFU with text rendering
+
+This rendering is a pure text report for every test run on all devices. It comes with some options:
+
+- Search (`--search`) for a regexp pattern in hostname and test name
+- Option to skip (`--skip-error`) tests in error (not failure) because of a connectivity issue or unsupported command
+
+Example output
+
+```bash
+$ anta nrfu text --tags pod1 --catalog nrfu/leaf.yml
+leaf2 :: verify_mlag_status :: SUCCESS
+leaf2 :: verify_mlag_interfaces :: SUCCESS
+leaf2 :: verify_mlag_config_sanity :: SUCCESS
+leaf2 :: verify_interface_utilization :: SUCCESS
+leaf2 :: verify_interface_errors :: SUCCESS
+leaf2 :: verify_interface_discards :: SUCCESS
+leaf2 :: verify_interface_errdisabled :: SUCCESS
+leaf2 :: verify_interfaces_status :: SUCCESS
+leaf2 :: verify_storm_control_drops :: SKIPPED (verify_storm_control_drops test is not supported on cEOSLab.)
+leaf2 :: verify_portchannels :: SUCCESS
+leaf2 :: verify_illegal_lacp :: SUCCESS
+leaf2 :: verify_loopback_count :: FAILURE (Found 3 Loopbacks when expecting 2)
+leaf2 :: verify_svi :: SUCCESS
+[...]
+```
+
+## NRFU with table report
+
+This rendering print results in a nice table suppoting grep filtering. It comes with its own set of options:
+
+- Search (`--search`) for a pattern in hostname and test name.
```

Comment:
in the `text` section you mention `regex` and here `pattern` for the `search` option, is it the same?

Reply:
Yes it is the same. If not clear I can change `regex` by `regex pattern`